desafio-10/desafio-10.css

```diff
@@ -0,0 +1,47 @@
+body{
+  background-color: #d9d9d9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  font-family: "Marker Felt", sans-serif;
+  font-size: 20px;
+}
+header {
```

Comment:
I think isn’t There’s a tag `header` in this Html file 🤔

Reply:
I removed from CSS, tks <3